content/articles/object-oriented-programming-in-python/index.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: engineering-education
+status: unpublish
+published: false
+url: /object-oriented-programming-in-python/
+title: Object Oriented Programming in Python
+description: This tutorial would guide the reader about object-oriented programming principles using the python programming language.
+author: joseph-abiola
+date:
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+    - url: /engineering-education/object-oriented-programming-in-python/hero.jpg
+      alt: OOP Python example image
+---
+
+Many modern languages like Java, C# are Object-Oriented or have support for using the Object-oriented programming (OOP) paradigms. Python is not exempted.
+
+In this tutorial, you would learn how to use OOP with Python.
+
+### Prerequisites
+Python should be installed on your local machine. I recommend at least from version `3.7` or newer.
+
+Some knowledge of Python is also required.
+
+### Project Overview
+For this tutorial, this would be the project we'll be using to explain most of the OOP principles found in the Python programming language.
+
+```python

```

Comment:
Explain this using words and not code. You are introducing the reader to this concepts, so the reader doesn't understand the code.